contracts/TalentLayerService.sol

```diff
@@ -36,55 +36,50 @@ contract TalentLayerService is Initializable, ERC2771RecipientUpgradeable, UUPSU
 
     /// @notice Service information struct
     /// @param status the current status of a service
-    /// @param buyerId the talentLayerId of the buyer
-    /// @param sellerId the talentLayerId of the seller
-    /// @param initiatorId the talentLayerId of the user who initialized the service
-    /// @param serviceDataUri token Id to IPFS URI mapping
+    /// @param ownerId the talentLayerId of the buyer
+    /// @param acceptedProposalId the accepted proposal ID
+    /// @param dataUri token Id to IPFS URI mapping
     /// @param proposals all proposals for this service
-    /// @param countProposals the total number of proposal for this service
     /// @param transactionId the escrow transaction ID linked to the service
     /// @param platformId the platform ID on which the service was created
     struct Service {
         Status status;
-        uint256 buyerId;
-        uint256 sellerId;
-        uint256 initiatorId;
-        string serviceDataUri;
-        uint256 countProposals;
+        uint256 ownerId;
+        uint256 acceptedProposalId;
+        string dataUri;
         uint256 transactionId;
         uint256 platformId;
     }
 
     /// @notice Proposal information struct
     /// @param status the current status of a service
-    /// @param sellerId the talentLayerId of the seller
+    /// @param ownerId the talentLayerId of the seller
     /// @param rateToken the token choose for the payment
     /// @param rateAmount the amount of token chosen
-    /// @param proposalDataUri token Id to IPFS URI mapping
+    /// @param dataUri token Id to IPFS URI mapping
     struct Proposal {
         ProposalStatus status;
-        uint256 sellerId;
+        uint256 ownerId;
         address rateToken;
         uint256 rateAmount;
         uint16 platformId;
-        string proposalDataUri;
+        string dataUri;
     }
 
     // =========================== Events ==============================
 
     /// @notice Emitted after a new service is created
     /// @param id The service ID (incremental)
-    /// @param buyerId the talentLayerId of the buyer
-    /// @param sellerId the talentLayerId of the seller
-    /// @param initiatorId the talentLayerId of the user who initialized the service
+    /// @param ownerId the talentLayerId of the buyer
+    /// @param acceptedProposalId the talentLayerId of the seller
```

Comment:
remove acceptedProposalId, in an open service, the seller/acceptedProposalId is always 0 at the beginning, so no need to pass it
> remove it from the event and remove it from the function